db/seeds.rb

```diff
@@ -57,6 +57,14 @@
   password: 'helloworld'
   )
 
+  # Create an moderator user
+  admin = User.create!(
```

Comment:
Small detail, should probably change this local variable to `moderator` to be consistent with the rest of the seed.